### DISCUSSION TITLE
Cannot input DA of 1367 in PERF APPR page

### DISCUSSION THREAD

**Discussion by Nadelbaum** [flybywiresim/aircraft#2335] [Help] [answered]
I wonder if this was already a known issue (a quick search through GitHub issues with DA keyword didn't provide anything): I was flying to LGSR with RNAV RWY 15 chart. The DA (H) for RWY15 is 1367. When typing this to PERF APPR page (DA entry) I got an error back. If I used the MDA of 1490 (MDA entry) then all was fine.

Are the limits hard coded too low?

**Comment by MapleNAV-TCASFAULT** [comment]
Holy DH of 1367 is a lot, I believe its out of range too!

**Reply by donstim** [reply]
The range of DHs accepted by the MCDU is from 0 to 700. Are you sure you are not reading OCA(H), or just plain DA? A DH of 1367 feet isn't even worth publishing.

**Reply by Nadelbaum** [reply]
![image](https://user-images.githubusercontent.com/22868316/101727048-67858b80-3abc-11eb-8eb1-edbfe3b2c0e3.png)

**Reply by Nadelbaum** [reply]
If you have Navigraph subscription (or Jeppesen or any other) just check LGSR (Santorini). I also might misinterpret this.

**Reply by Nadelbaum** [reply]
Ok, I just realized, and please confirm, but DH would be only used for ILS approaches (Cat II/III), right? All other approaches should use the MDA field?

**Reply by MapleNAV-TCASFAULT** [reply]
Not sure but I think, MDA is used for ILS while DA/DH is for other types of approaches, depends on the type of terrain and approach.

**Reply by Nadelbaum** [reply]
Referring to these two sources:

http://www.a320dp.com/A320_DP/nav-flight-management/sys-14.5.49.html
http://www.a320dp.com/A320_DP/nav-flight-management/sys-14.5.50.html

MDA - Allows entry of barometric minimums by the pilot for all CAT I ILS/nonprecision approaches
DH - Allows entry of radio altitude minimums by the pilot for CAT II/III approaches.

Everyday is a new day for learning. Thanks Maple and donstim for your replies!

**Reply by MapleNAV-TCASFAULT** [reply]
Lol, I got it the other way around, but as you said “everyday is a new day for learning”.

**Reply by donstim** [reply]
Actually, it looks like you answered it yourself before I got my reply in!  I just didn't see the messages past your attachment of the approach chart until I replied. Give yourself the credit for answering your question! xD

**Reply by Nadelbaum** [reply]
I don't mind. You provided additional valuable information as well.

**Comment by donstim** [comment, marked as answer]
It is a non-precision approach, so it's giving you the MDA per approach category. For Category C airplanes, the MDA is 1490 feet, and for Category D airplane, MDA is 1690 feet. A minimum runway visual range of 5000 meters is required for both categories. The A320 is an approach Category C airplane.

The numbers in parentheses are for military airplanes and don't apply to civilian operations.
